postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -527,14 +546,7 @@ def _run_explain(self, dbname, statement, obfuscated_statement):
                 if not result or len(result) < 1 or len(result[0]) < 1:
                     return None
                 return result[0][0]
-            except Exception as e:
-                self._check.count(
-                    "dd.postgres.run_explain.error",
-                    1,
-                    tags=self._dbtags(dbname, "error:explain-database_error-{}".format(type(e)))
-                    + self._check._get_debug_tags(),
-                    hostname=self._check.resolved_hostname,
-                )
```

Comment:
We were emitting a hard coded error code for our `dd.postgres.run_explain.error ` metric, so this change now assigns the appropriate error code.